companion.py

```diff
@@ -4,32 +4,35 @@
 import base64
 import csv
 import requests
-from http.cookiejar import LWPCookieJar	# No longer needed but retained in case plugins use it
+from http.cookiejar import LWPCookieJar  # No longer needed but retained in case plugins use it
 from email.utils import parsedate
 import hashlib
-import json
 import numbers
 import os
-from os.path import dirname, isfile, join
+from os.path import join
 import random
 import sys
 import time
 from traceback import print_exc
 import urllib.parse
 import webbrowser
-import zlib
 
 from config import appname, appversion, config
 from protocol import protocolhandler
 
+from typing import TYPE_CHECKING
```

Comment:
Dont forget to remove this before merge

Reply:
As this is essentially just doing `TYPE_CHECKING = False` I don't see the harm in leaving it in.  If/when we make more use of `typing` we can address it then.

Reply:
True, my main concern is that Im not entirely sure how _ is hacked in, and whether or not there is any checking on whether or not there are any assignments ever. during normal runtime TYPE_CHECKING should always be false

Reply:
Well, running this branch and switching to French appears to still be translating everything without throwing exceptions or errors.

Reply:
Ah, but in companion.py itself the only use of `_()` is in errors that I haven't triggered yet.  Time to add something temporarily to be sure.

Reply:
`print('Testing translation: "{KEY}" -> "{TRANSLATED}"'.format(KEY='Error: Wrong Cmdr', TRANSLATED=_('Error: Wrong Cmdr')))` does correctly yield `Testing translation: "Error: Wrong Cmdr" -> "Erreur : Cmdr incorrect"` when placed in the Session.login() method, right at the start.

Reply:
Cool, then leaving it in should be fine, will do so